src/preact10-rst.ts

```diff
@@ -42,7 +42,10 @@ function convertDOMProps(props: Props) {
 /**
  * Convert the rendered output of a vnode to RST nodes.
  */
-function rstNodesFromChildren(nodes: (VNode | null)[] | null): RSTNodeTypes[] {
+function rstNodesFromChildren(
+  nodes: (VNode | null)[] | null,
+  isShallow: boolean
```

Comment:
I'm wondering whether the `isShallow` flag is better named after the context where it is used (shallow rendering) or what it does (preserve fragments in the output). Currently preserving fragments is the only thing it does, so that might be a more obvious name. Are there any other behaviors that might need to change when shallow rendering?

Reply:
Hmm yea good call. I've renamed to `preserveFragments`. If we discover we end up having more options related to shallow rendering, perhaps we consider `isShallow` then. But hopefully we don't have more options lol and so `preserveFragments` is clearer.